runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/WesternDiaryRequirement.java

```diff
@@ -72,7 +76,9 @@ public WesternDiaryRequirement()
 			new QuestRequirement(Quest.ONE_SMALL_FAVOUR),
 			new QuestRequirement(Quest.THE_GRAND_TREE));
 		add("Claim a Chompy bird hat from Rantz after registering at least 125 kills.",
-			new QuestRequirement(Quest.BIG_CHOMPY_BIRD_HUNTING));
+			new QuestRequirement(Quest.BIG_CHOMPY_BIRD_HUNTING),
+			new SkillRequirement(Skill.FLETCHING, 5),
```

Comment:
Same for these, they are already part of the quest requirement.

Reply:
That's a good point, I removed that commit.